tests/scripts/t3000/run_t3000_model_perf_tests.sh

```diff
@@ -44,6 +44,20 @@ run_t3000_llama2_70b_tests() {
   echo "LOG_METAL: run_t3000_llama2_70b_tests $duration seconds to complete"
 }
 
+run_t3000_falcon40b_tests() {
+  # Record the start time
+  start_time=$(date +%s)
+
+  echo "LOG_METAL: Running run_t3000_falcon40b_tests"
+
+  env WH_ARCH_YAML=wormhole_b0_80_arch_eth_dispatch.yaml pytest models/demos/t3000/falcon40b/tests/test_perf_falcon.py -m "model_perf_t3000"
```

Comment:
I don' think Falcon40 was run in the posted run, is "model_perf_t3000" incorrect here?